modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/test/clazz/group/ModulesJUnitBatchTestClassGroup.java

```diff
@@ -129,11 +131,10 @@ protected List<JobProperty> getRelevantIncludesJobProperties() {
 			return super.getRelevantIncludesJobProperties();
 		}
 
-		List<File> modifiedFilesList =
-			portalGitWorkingDirectory.getModifiedFilesList();
+		Set<File> modifiedModuleDirsList = new HashSet<>();
```

Comment:
@brittneyq modifiedModuleDirsList is a Set, not a List.

Reply:
oops sorry I'll make that change